### ISSUE TITLE
Cannot install XCode command line tools due to "perl: realpath couldn't resolve"

### ISSUE THREAD

**Issue by mrolappe** [darlinghq/darling#1112]
**Expected Result**
I expected the installation to complete successfully.

**Actual Result**
The installation aborted with an error message "perl: realpath couldn't resolve..."

**Steps To Reproduce**
1. Download Command_Line_Tools_for_Xcode_12.dmg
2. Mount disk image
3. Start installation from disk image

```
Darling [/Volumes/SystemRoot/mnt/freiraum/darling/darling/build-darlingserver]$ hdiutil attach /Volumes/SystemRoot/mnt/freiraum/darling/Command_Line_Tools_for_Xcode_12.dmg 
/Volumes/Command_Line_Tools_for_Xcode_12
Darling [/Volumes/SystemRoot/mnt/freiraum/darling/darling/build-darlingserver]$ cd /Volumes/Command_Line_Tools_for_Xcode_12
Darling [/Volumes/Command_Line_Tools_for_Xcode_12]$ ls
Command Line Tools.pkg
Darling [/Volumes/Command_Line_Tools_for_Xcode_12]$ installer -pkg Command\ Line\ Tools.pkg -target /
installer: Package name is SU_TITLE
installer: Installing at base path /
installer: Installing selected choice SU_TITLE
installer: Installing package com.apple.pkg.CLTools_Executables version 12.0.0.0.1.1599194153 (1600651 KB)
installer: Extracting files
perl: realpath couldn't resolve "/home/mrolappe/.darling/usr/bin/perl"
The script has terminated with exit code 1

```

**System Information**
Using `darlingserver` branch on Fedora 35.

| Software | Version |
| --- | --- |
| Linux Kernel | 5.16.11 |
| Darling | c30e22f13ea61d867d76adc2914ec8352cac66be |

**Comment by facekapow** [comment]
This is fixed in e8a1f0e3d08cce51dda7e4051b1b4529f25c118c.

**Comment by mrolappe** [comment]
Great, thanks.
